routes/products.js

```diff
@@ -0,0 +1,108 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database');
+
+router.route('/').get((req, res) => {
+  knex
+    .raw('select * from products')
+    .then((productObject) => {
+      return res.status(200).send(productObject.rows);
+    })
+    .catch((err) => {
+      return res.status(500).send("{ 'message' : 'Database error' }");
+    });
+});
+
+router
+  .route('/:product_id')
+  .get((req, res) => {
+    knex
+      .raw('select * from products where id = ?', [req.params.product_id])
+      .then((productObject) => {
+        if (productObject.rows.length === 0) {
+          return res.status(404).send('{ "message": "Product not found" }');
+        }
+
+        return res.status(200).send(productObject.rows);
```

Comment:
You should return the product object itself instead of an array of just one object.